packages/sak-visittkort/src/VisittkortSakIndex.stories.tsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import { StoryFn } from '@storybook/react';
 
+import moment from 'moment/moment';
```

Comment:
dayjs i staden for moment (Moment er framleis i bruk rundt omkring, men har du tid så skriv om til dayjs)